g11n-ws/modules/md-data-api-s3impl/src/main/java/com/vmware/vip/messages/data/dao/impl/S3ProductDaoImpl.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vip.messages.data.dao.impl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import com.amazonaws.services.s3.model.ListObjectsV2Result;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import com.vmware.vip.common.constants.ConstantsChar;
+import com.vmware.vip.common.constants.ConstantsFile;
+import com.vmware.vip.messages.data.conf.S3Cient;
+import com.vmware.vip.messages.data.conf.S3Config;
+import com.vmware.vip.messages.data.dao.api.IProductDao;
+import com.vmware.vip.messages.data.dao.exception.DataException;
+import com.vmware.vip.messages.data.util.S3Utils;
+
+/**
+ * this class use to get the properties of a version bundle files
+ */
+@Repository
+public class S3ProductDaoImpl implements IProductDao {
+
+   private static Logger logger = LoggerFactory.getLogger(S3ProductDaoImpl.class);
+
+   @Autowired
+   private S3Cient s3Client;
```

Comment:
it should be removed since it's a global singleton object now, or it should be changed to same usage of above 'logger'.

Reply:
we use the spring manage the object， the spring keep the object is singleton